tezos/api/src/ffi.rs

```diff
@@ -751,6 +752,58 @@ pub struct RpcRequest {
     pub accept: Option<String>,
 }
 
+fn ffi_routed_subpath(path: &str) -> String {
+    let base = Url::parse("http://tezedge.com")
+        .expect("[Unexpected] cannot parse a well formed example URL");
+    let parsed = Url::options().base_url(Some(&base)).parse(path).unwrap();
+    let normalized_path = match parsed.query() {
+        Some(query) => format!("{}?{}", parsed.path().trim_end_matches('/'), query),
+        None => parsed.path().trim_end_matches('/').to_string(),
+    };
+    let mut segments = match parsed.path_segments() {
+        Some(segments) => segments,
+        // Not the subpath we expect, bail-out
+        None => return normalized_path,
+    };
+
+    // /chain/:chain_id/blocks/:block_id
+    let (chain, _, blocks, _) = (
+        segments.next(),
+        segments.next(),
+        segments.next(),
+        segments.next(),
+    );
+
+    match (chain, blocks) {
+        (Some("chain"), Some("blocks")) => (),
```

Comment:
@tizoc  the subpath is suppose to be `chains/<chain_id>/blocks/<block_id>`, so it should be 
```
(Some("chains"), Some("blocks"))
```

Reply:
https://tezos.gitlab.io/shell/rpc.html

Reply:
fixed, good catch